go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/bsm/macdaddy
 
-go 1.15
+go 1.18
 
 require (
-	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
-	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
+	github.com/bsm/ginkgo v1.16.5
+	github.com/bsm/gomega v1.27.8-0.20230530072558-a9d61a92ed40
```

Comment:
go mod didnt like the `27.7.1.1` latest tag btw, compalined and then `go mod tidy` auto corrected it to `1.27.8-0`

Reply:
Hmm, I will see if I can fix this. In the meantime, please upgrade to ginkgo/v2

Reply:
this should now be updated you can `go get -u github.com/bsm/gomega`